.github/workflows/reusable_testing.yml

```diff
@@ -44,3 +51,6 @@ jobs:
 
       - run: cmake . && make
       - run: make test-memcached
+        # TODO: Remove this when https://github.com/tarantool/memcached/issues/96
+        # is resolved.
+        if: ${{ !startsWith(env.TNT_VERSION, 'Tarantool 2.10') }}
```

Comment:
The version is actually 2.10+, not only 2.10.x.

Reply:
It is not easy to express using workflows expression syntax and I very hope that we'll fix it prior 2.11 will ever start.

Reply:
We can use the following condition:
```
        if: ${{ startsWith(env.TNT_VERSION, 'Tarantool 1.10') || 
          startsWith(env.TNT_VERSION, 'Tarantool 2.8') }}
```

Reply:
It is not correct. Bug exists in versions where:
- MAJOR version is equal to 2 *and* MINOR version is bigger or equal to 8.
or
- MAJOR version is bigger than 2

Thus, your condition will be incorrect for string 'Tarantool 3.1.x'.

Reply:
The bug doesn't exist for tarantool 2.8. Please check CI jobs in the `master` branch to make sure.

Reply:
Okey, now I got it.